src/router/index.js.bak

```diff
@@ -0,0 +1,121 @@
+import { createWebHistory, createRouter } from 'vue-router';
```

Comment:
Why is this file here?

Reply:
This has been removed. Please check again